demo/screens/Home.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Image, Button, StyleSheet, View, Text } from "react-native";
+import { createStackNavigator, createAppContainer } from "react-navigation";
+import { Account } from "tasit-sdk";
+import ListLands from "./ListLands";
+import LandClaim from "./LandClaim";
+import OnboardingHome from "./OnboardingHome";
+import EthereumQuestion from "./EthereumQuestion";
+import EthereumSignUp from "./EthereumSignUp";
+import EthereumSignIn from "./EthereumSignIn";
+
+class Home extends React.Component {
+  componentDidMount = async () => {
+    let wallet = await Account.create();
```

Comment:
@marcelomorgado We decided to remove the await, right?

Reply:
Yes but it's still using tasit-sdk 0.0.2, I'll make this change on new publish.

Reply:
Good point. Sounds good. Sometimes to keep things like this in the "organizational memory" even if we both forget, I like putting something like a `// TODO: Remove await when SDK 0.0.3 is out` comment in the code

Reply:
Also, I created a new issue that this reminded me of https://github.com/tasitlabs/TasitSDK/issues/52

Reply:
OK, noted!